spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
```

Comment:
I understand that this could be done however in the same way as I prefer having self around. I prefer to have the FactoryGirl around so it is clear what is being called.